products/template/build_scripts/__init__.py

```diff
@@ -15,7 +15,6 @@
 PRODUCT_S3_NAME = "db-template"
```

Comment:
Leftover from me cleaning up dcpy installation - can you ✂️ line 13 of this file?